File: test-jwt.js
const { jwtVerify } = require('jose');
const jwt = require('jsonwebtoken');

const secret = new TextEncoder().encode("super-secret-default-key-1234");
const token = jwt.sign({ sub: 'test', exp: Math.floor(Date.now() / 1000) + 3600 }, "super-secret-default-key-1234", { algorithm: 'HS256' });

async function run() {
  try {
    const verified = await jwtVerify(token, secret);
    console.log("Verified:", verified.payload);
  } catch (e) {
    console.error("Error:", e);
  }
}
run();

File: test_jwt.py
import os
from datetime import datetime, timedelta
import jwt

SECRET_KEY = "super-secret-default-key-1234"
ALGORITHM = "HS256"
access_token_expires = timedelta(hours=24)
expire = datetime.utcnow() + access_token_expires

to_encode = {
    "sub": "1",
    "username": "admin",
    "role": "admin",
    "api_key": "some-api-key",
    "exp": expire,
}
encoded_jwt = jwt.encode(to_encode, SECRET_KEY, algorithm=ALGORITHM)
print(encoded_jwt)
